lib/config/presets/index.ts

```diff
@@ -138,6 +142,9 @@ export function parsePreset(input: string): ParsedPreset {
       .map((elem) => elem.trim());
     str = str.slice(0, str.indexOf('('));
   }
+  if (presetSource === 'http') {
+    return { presetSource, repo: str, presetName: 'default', params };
```

Comment:
```suggestion
    return { presetSource, repo: str, presetName: '', params };
```

Is `presetName` required? then we should probably set it to empty string to show it's not used